docs/components/MachineLearning.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_title: "Machine Learning"
```

Comment:
The same here - can u add ` (Enterprise only)` suffix?

Reply:
or maybe put this docs in components/enterprise directory